{
  "cveId": "CVE-2019-25164",
  "eventName": "New CVE Received",
  "cveChangeId": "8949D8ED-D3D2-4678-B3DF-8E38BF91692B",
  "sourceIdentifier": "416baaa9-dc9f-4396-8d5f-8c081fb06d67",
  "created": "2025-09-16T16:15:33.567",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "Rejected reason: ** REJECT ** DO NOT USE THIS CVE RECORD. ConsultIDs: none. Reason: This record was in a CNA pool that was not assigned to any issues during 2019. Notes: none."
    }
  ]
}